deepgram/transcription.py

```diff
@@ -64,6 +119,19 @@ async def __call__(self) -> 'LiveTranscription':
         return self
 
     async def _start(self) -> None:
+        """
+        The _start function is the main function of the LiveTranscription class.
+        It is responsible for creating a websocket connection to Deepgram Transcribe,
+        and then listening for incoming messages from that socket. It also sends any 
+        messages that are in its queue (which is populated by other functions). The 
+        _start function will run until it receives a message with an empty transcription, 
+        at which point it will close the socket and return.
+        
+        :param self: Used to Access the class attributes.
```

Comment:
Please remove this line from docstring `:param self: Used to Access the class attributes.`